docs/dev/table/connectors/jdbc.md

```diff
@@ -0,0 +1,395 @@
+      <td>optional</td>
+      <td style="word-wrap: break-word;">3</td>
+      <td>Integer</td>
+      <td>The max retry times if writing records to database failed.</td>
+    </tr>          
+    </tbody>
+</table>
+
+Data Type Mapping
+----------------
+Flink support connect to several JDBC dialect，the field type mappings between Flink and current supported databases are as follows:
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left">Flink Data Type</th>
+        <th class="text-left">MySQL Type</th>
+        <th class="text-left">PostgreSQL Type</th>
+        <th class="text-left">Derby Type</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>BOOLEAN</td>
+      <td>TINYINT</td>
+      <td>BOOLEAN</td>
+      <td>BOOLEAN</td>
+    </tr>
+    <tr>
+      <td>TINYINT</td>
+      <td>TINYINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+    </tr>
+    <tr>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+      <td>SMALLINT</td>
+    </tr>
+    <tr>
+      <td>INT</td>
+      <td>INT</td>
+      <td>INT</td>
+      <td>INT</td>
+    </tr>
+    <tr>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+      <td>BIGINT</td>
+    </tr>
+    <tr>
+      <td>FLOAT</td>
+      <td>FLOAT</td>
+      <td>REAL</td>
+      <td>FLOAT</td>
+    </tr>
+    <tr>
+      <td>DOUBLE</td>
+      <td>DOUBLE</td>
+      <td>DOUBLE PRECISION</td>
+      <td>DOUBLE PRECISION</td>
+    </tr>
+    <tr>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+      <td>DECIMAL</td>
+    </tr>
+    <tr>
+      <td>CHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+    </tr>
+    <tr>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+    </tr>
+    <tr>
+      <td>STRING</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+      <td>VARCHAR</td>
+    </tr>
+    <tr>
+      <td>DATE</td>
+      <td>DATE</td>
+      <td>DATE</td>
+      <td>DATE</td>
+    </tr>
+    <tr>
+      <td>TIME</td>
+      <td>TIME</td>
+      <td>TIME</td>
+      <td>TIME</td>
+    </tr>
+    <tr>
+      <td>TIMESTAMP</td>
+      <td>TIMESTAMP</td>
+      <td>TIMESTAMP</td>
+      <td>TIMESTAMP</td>
+    </tr>
+    <tr>
+      <td>ARRAY</td>
+      <td>Unsupported</td>
+      <td>ARRAY</td>
+      <td>Unsupported</td>
+    </tr>
+    <tr>
+      <td>BINARY</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+    </tr>
+    <tr>
+      <td>VARBINARY</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+    </tr>    
+    <tr>
+      <td>MAP/MULTISET</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+    </tr>
+    <tr>
+      <td>ROW</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+      <td>Unsupported</td>
+    </tr>                     
+    </tbody>
+</table>
```

Comment:
Could you add links about the types supported by these databases? 
For example, postgres: https://www.postgresql.org/docs/12/datatype.html